client/sharkey_client.go

```diff
@@ -80,11 +81,13 @@ func main() {
 		log.Print("Generated http client")
 	}
 
-	if c.conf.Sleep == "" {
+	if c.conf.Sleep == "" || c.conf.PingOnStart {
```

Comment:
We should just remove this if statement.

I don't think it was intentional to wait the sleep duration at startup -- Seems like a bug introduced when we switched from sleeping in a loop to using the ticker.